client/src/components/Account/UploadPhoto/UploadPhoto.tsx

```diff
@@ -1,12 +1,88 @@
-import { Box, Typography } from '@material-ui/core';
+import { Box, Typography, Avatar, Grid, Button, FormControl } from '@material-ui/core';
+import DeleteIcon from '@material-ui/icons/Delete';
+import SaveIcon from '@material-ui/icons/Save';
+
+import React, { ChangeEventHandler, useRef, useState } from 'react';
 
 import useStyles from './useStyles';
+import profile_url from '../../../Images/b1f0e680702e811aa8ba333cb19c0e0ea95e8e31.png';
+import { isNull } from 'util';
+
+//this image here is just for mockup, will be updated in
+//the future when amazon s3 links are avialible
 
 export default function UploadPhoto(): JSX.Element {
   const classes = useStyles();
+  const inputFile = useRef<HTMLInputElement>(null);
+  const onButtonClick = () => {
+    if (inputFile.current !== null) {
+      inputFile.current.click();
+    }
+  };
+  const [imageUrl, setImageUrl] = useState<string>(profile_url);
+
+  const handleImageChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    e.preventDefault();
+    const reader = new FileReader();
+    e.target.files instanceof FileList ? reader.readAsDataURL(e.target.files[0]) : 'handle exception';
+
+    reader.onloadend = () => {
+      if (reader.result) {
+        setImageUrl(reader.result as string);
+      }
+    };
+  };
+
   return (
-    <Box className={classes.container}>
-      <Typography variant="h3">Upload Photo</Typography>
+    <Box>
+      <Typography variant="h4" align="center" className={classes.formTitle}>
+        Profile Photo
+      </Typography>
+      <Grid container direction="column" alignItems="center" spacing={4}>
+        <Grid item>
+          <Avatar alt="Profile picture" src={imageUrl} className={classes.large} />
+        </Grid>
+        <Grid item>
+          <Typography align="center" color="secondary" className={classes.uploadMessage}>
+            Be sure to use a photo that clearly shows your face
+          </Typography>
+        </Grid>
+        <Grid item>
+          <FormControl>
+            <Grid container direction="column" spacing={3}>
+              <Grid item>
+                <Button color="primary" variant="outlined" size="large" onClick={onButtonClick}>
+                  <Box px={2} py={1}>
+                    <input
+                      type="file"
+                      id="file"
+                      ref={inputFile}
+                      style={{ display: 'none' }}
+                      required
+                      onChange={(e: React.ChangeEvent<HTMLInputElement>): void => handleImageChange(e)}
+                    />
+                    Upload a file from your device
+                  </Box>
+                </Button>
+              </Grid>
+              <Grid item>
+                <Grid container direction="row" justify="space-around">
+                  <Grid>
+                    <Button color="secondary" startIcon={<SaveIcon style={{ color: 'black' }} />}>
+                      Save Photo
+                    </Button>
```

Comment:
I think this button is adding an extra step that is weird from a UX perspective. If it's to match the mocks then you can ignore my comment for now and figure out later in implementation. If you do keep it, I think it should change color or something when you've uploaded the image so the user knows to click it to complete the update

Reply:
No problem, I deleted the save button